src/changes.rs

```diff
@@ -166,15 +188,30 @@ impl<'a> ChangeSet<'a> {
             }
             WriteMode::NewFile(extn) => {
                 let filename = filename.to_owned() + "." + extn;
-                let mut file = try!(File::create(&filename));
-                try!(write!(file, "{}", text));
+                let file = try!(File::create(&filename));
+                try!(style(file, text));
             }
             WriteMode::Display => {
                 println!("{}:\n", filename);
-                println!("{}", text);
+                let lock = stdout();
```

Comment:
this lock variable should have a better name, I think, its not clear what is getting locked. Perhaps `stdout` and `stdout_writer` or something?